docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md

```diff
@@ -0,0 +1,110 @@
+# Dasharo Compatibility: Suspend mechanism switching (S0ix/S3)
```

Comment:
@johanes2115 this is not the part of our task. Please clean up the branches, on which you are working.

Reply:
@sulewskiprzemyslaw I deleted the file from the commit